src/EditorFeatures/Core/InlineHints/InlineParameterNameHintsDataTaggerProvider.cs

```diff
@@ -54,7 +53,10 @@ protected override ITaggerEventSource CreateEventSource(ITextView textViewOpt, I
             // TaggerDelay is NearImmediate because we want the renaming and tag creation to be instantaneous
             return TaggerEventSources.Compose(
                 TaggerEventSources.OnViewSpanChanged(ThreadingContext, textViewOpt, textChangeDelay: TaggerDelay.Short, scrollChangeDelay: TaggerDelay.NearImmediate),
-                TaggerEventSources.OnWorkspaceChanged(subjectBuffer, TaggerDelay.NearImmediate, _listener));
+                TaggerEventSources.OnWorkspaceChanged(subjectBuffer, TaggerDelay.NearImmediate, _listener),
+                TaggerEventSources.OnOptionChanged(subjectBuffer, InlineHintsOptions.ForLiteralParameters, TaggerDelay.NearImmediate),
```

Comment:
I believe the PerLanguageOptions collection on line 37ish is used by the tagger to automatically create the OnOptionChanged for you; I think you can just update that collection and this work is done for you. In either case we should just be consistent.

Reply:
that's not quite how it works.  the taggers have options that completely enable/disable them.   then there are the triggers that ask if we should recompute tags.

So the split here is appropriate.  teh one on line 37 determiens if we run at all.  these indicate when we should recompute tags.  I will doc :)

Reply:
nope.  i'm wrong.  i misread.  fixing this up.

Reply:
no.  i was right hte first time. we skip production if any of these optoins are disabled.  reverting.